source/mongodump/mongodump-behavior.txt

```diff
@@ -209,6 +209,14 @@ following limitations apply:
   - :option:`--dumpDbUsersAndRoles`
   - :option:`--oplog <mongodump --oplog>`
 
+.. note:: Rollback on the target cluster during a restore or import
+   
+   If you are restoring or importing data to a cluster that undergoes a
+   rollback during the restore or import process, you should delete all 
```

Comment:
Remove the 'you should', since it's probably softer than what we want to say.

Reply:
I agree – fixed!